src/Osma.Mobile.App/Views/Connections/AcceptInvitePage.xaml

```diff
@@ -49,18 +49,18 @@
                 Orientation="Horizontal"
                 HorizontalOptions="FillAndExpand"
                 VerticalOptions="FillAndExpand">
-                <Button
-                    HorizontalOptions="FillAndExpand"
-                    Style="{DynamicResource BlueButtonStyle}"
-                    IsEnabled="{Binding IsBusy, Converter={StaticResource BooleanInverse}}"
-                    Command="{Binding AcceptInviteCommand}"
-                    Text="Trust"/>
                 <Button
                     Style="{DynamicResource RedButtonStyle}"
                     IsEnabled="{Binding IsBusy, Converter={StaticResource BooleanInverse}}"
                     Command="{Binding RejectInviteCommand}"
                     HorizontalOptions="FillAndExpand"
-                    Text="Dont trust"/>
+                    Text="cancel"/>
```

Comment:
I think this should be upper case 'Cancel'

Reply:
Agreed, I just updated them.